client/asset/eth/eth.go

```diff
@@ -407,6 +412,17 @@ func (m *monitoredTx) UnmarshalBinary(data []byte) error {
 	return nil
 }
 
+// pendingTx is used to track unconfirmed transaction that should be considered
+// for balance calculations, for node types that don't support viewing mempool
+// transactions directly.
+type pendingTx struct {
+	assetID   uint32
+	out       uint64 // eth or token
+	in        uint64 // eth or token
+	fees      uint64 // eth
```

Comment:
These `fees` gotta be `maxFee*gasLimit` right ? Might call it `maxFee` to clarify that.